etc/kayobe/ansible/fix-houston.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Create Systemd Unit to workaround 'tc mirred to Houston' error
+  hosts: network,compute
+  become: yes
+
+  tasks:
+    - name: Include kolla-ansible host vars
+      include_vars: "{{ kolla_config_path }}/inventory/overcloud/host_vars/{{ inventory_hostname }}"
```

Comment:
Probably this fix belongs in kolla-ansible. Is there at least a bug report for it?

Reply:
That only happens when we use Mellanox ASAP2 if I remember correctly, it rather belongs to Mellanox/NVIDIA

Reply:
Ok, might be helpful to mention that in the release note and add a comment in the playbook about when to use it.

Reply:
done